.github/workflows/deploy.yml

```diff
@@ -1,46 +1,34 @@
-# This file was created automatically with `myst init --gh-pages` 🪄 💚
-
 name: MyST GitHub Pages Deploy
```

Comment:
I would call make this short and call this workflow "Documentation". That will result in more descriptive stages

- Documentation / build
- Documentation / deploy

![image](https://github.com/ComPWA/gluex-nstar/assets/29308176/962a3ca2-988f-4952-aa1a-727bcbecfe4d)

It's also more general, so that if you switch to another build system than MyST (like Quarto), the workflow name does not have to be changed.

Reply:
Nice suggestion!